unit_tests/test_nemesis.py

```diff
@@ -164,9 +171,23 @@ def test_disabled_monkey():
     all_disrupt_methods = {attr[1].__name__ for attr in inspect.getmembers(Nemesis) if
                            attr[0].startswith('disrupt_') and
                            callable(attr[1])}
+    tester.params["nemesis_exclude_disabled"] = True
+    sisyphus = FakeSisyphusMonkey(tester, None)
 
-    tester.params["nemesis_selector"] = ['!disabled']
-    sisphus = FakeSisyphusMonkey(tester, None)
-
-    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisphus.disruptions_list}
+    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
+    # Note: this test will fail and have to be adjusted once additional 'disabled' nemeses added.
     assert collected_disrupt_methods_names == all_disrupt_methods - {'disrupt_toggle_table_gc_mode'}
+
+
+def test_use_disabled_monkey():
+
+    ToggleGcModeMonkey.disabled = True
+
+    tester = FakeTester()
+
+    tester.params["nemesis_exclude_disabled"] = False
+    sisyphus = FakeSisyphusMonkey(tester, None)
+
+    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
+
+    assert 'disrupt_toggle_table_gc_mode' in collected_disrupt_methods_names
```

Comment:
we should assert like in previous test using all_disrupt methods:
`assert collected_disrupt_methods_names == all_disrupt_methods`
so in case it was removing some other method it would fail.

Reply:
basically you're right, but i think it is already covered in above tests so no need to repeat it here. this one is focusing in the minimal validation of using a disabled nemesis.